docs/book/src/tasks/certs/auto-rotate-certificates-in-kcp.md

```diff
@@ -0,0 +1,72 @@
+## Automatically rotating certificates using Kubeadm Control Plane provider
```

Comment:
Just a blocker comment with notes for now. I'll update depending on the outcome of some discussions and the other PR:
* We should document that we assume all certificates are generated with the same expiry
* [TBD] We should document that a manual certificate rotation is only complete after the kube-apiserver is serving the new certs

(only applies if the new PR is merged)

Reply:
>We should document that a manual certificate rotation is only complete after the kube-apiserver is serving the new certs and the expiry annotation has been removed from KubeadmConfig

The annotation will be added back on the new KubeadmConfig by KCP, isnt it?

Reply:
Added a suggestion below for a warning